main.py

```diff
@@ -1,10 +1,53 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+    Player.objects.all().delete()
+
+    with open("players.json") as f:
+        players = json.load(f)
+
+    for player in players:
+        race_name = players[player]["race"]["name"]
+        race_description = players[player]["race"]["description"]
+        race, _ = Race.objects.get_or_create(
+            name=race_name,
+            description=race_description
+        )
```

Comment:
Do not create these variables, pass them to the method
```suggestion
        race, _ = Race.objects.get_or_create(
            name=player_data["race"]["name"],
            description=player_data["race"]["description"]
        )
```